### PR TITLE
Custom icons in NeumorphicAppBar + examples

### PR DESCRIPTION
**Pull request by Overman775**

### CODE REVIEW

File: lib/src/widget/back_button.dart [hunk 1]
```diff
@@ -6,36 +8,28 @@ class NeumorphicBackButton extends StatelessWidget {
...
   Widget build(BuildContext context) {
-    final theme = Theme.of(context);
+    final nThemeIcons = NeumorphicTheme.of(context).current.appBarTheme.icons;
     return NeumorphicButton(
       style: style,
       padding: padding,
       tooltip: MaterialLocalizations.of(context).backButtonTooltip,
-      child: Icon(_getIcon(theme)),
+      child: reversedIcon
+          ? Transform.rotate(
```
Comment (jaumard) [inline lib/src/widget/back_button.dart:29]: a bit overkill to introduce transformation for that imho.
Why not adding a `forward` icon instead into your `NeumorphicAppBarIcons` the idea was to have back and forward if needed

File: lib/src/widget/close_button.dart [hunk 2]
```diff
@@ -6,23 +8,27 @@ class NeumorphicCloseButton extends StatelessWidget {
...
+    final nThemeIcons = NeumorphicTheme.of(context).current.appBarTheme.icons;
     return NeumorphicButton(
       style: style,
       padding: padding,
       tooltip: MaterialLocalizations.of(context).closeButtonTooltip,
-      child: Icon(
-        Icons.close,
-      ),
+      child: reversedIcon
+          ? Transform.rotate(
```
Comment (jaumard) [inline lib/src/widget/close_button.dart:29]: same feeling here not a fan of adding transformation for that... let's keep it simple of if people need more they can just do it in their app. Not sure a close button will ever need to be reversed.

File: lib/src/widget/back_button.dart (hunk 1 above)
Comment (Overman775) [reply lib/src/widget/back_button.dart:29]: Ok, I will do `forward` icon in `NeumorphicAppBarIcons` and delete `Transform.rotate`, then need rename `bool reversedIcon` to `forward`

File: lib/src/widget/close_button.dart (hunk 2 above)
Comment (Overman775) [reply lib/src/widget/close_button.dart:29]: Yep, need delete

**Comment by florent37** [comment]
Thanks that really great

Do you want me to ship a 3.0.1 or you have others things to fix/add ?

**Comment by florent37** [comment]
I think to add gihub action to publish version of the number change :)

**Comment by Overman775** [comment]
> Do you want me to ship a 3.0.1 or you have others things to fix/add ?

There are no ideas yet, if new ideas appear I will do another PR

**Comment by florent37** [comment]
I added the GitHub action, if you update the version in changelog & pubspec, then publish to master (via a merge resquest) it will publish the version to pub.dev

I tested it with 3.0.1 :)

**Pull request merged**